File: Python to Dalvik Compiler/CodigosTeste/micro08/micro08.py
def micro08():
	numero =1
	while numero < 0 or numero >0:
		print("Digite o numero",end="")
		numero = int(input())
		if numero > 10:
			print("O numero "+str(numero)+" e maior que 10")
		else:
			print("O numero "+str(numero)+" e menor que 10")


micro08()

File: Python to Dalvik Compiler/CodigosTeste/micro08/micro08.java
import java.util.Scanner;

public class micro08
{
	public static void main(String[] args)
	{
		Scanner s = new Scanner(System.in);
		int numero =1;
		while (numero < 0 || numero >0){
			System.out.print("Digite o numero");
			numero = s.nextInt();
			if (numero > 10)
				System.out.println("O numero "+numero+" e maior que 10");
			else
				System.out.println("O numero "+numero+" e menor que 10");
		}
	}
}